src/QuickMarcEditor/utils.js

```diff
@@ -807,3 +807,37 @@ export const splitFields = marcRecord => {
     }),
   };
 };
+
+export const are010Or1xxUpdated = (initial, updated) => {
+  let is010Updated = false;
+  let is1XXUpdated = false;
+
+  const initial010 = initial.find(rec => rec.tag === '010');
+  const updated010 = updated.find(rec => rec.tag === '010');
+
+  if (initial010 &&
+    updated010 &&
+    getContentSubfieldValue(initial010.content).$a !== getContentSubfieldValue(updated010.content).$a
+  ) {
+    is010Updated = true;
+  }
+
+  const initial1xxRecords = initial.filter(rec => rec.tag[0] === '1');
+  const updated1xxRecords = updated.filter(rec => rec.tag[0] === '1');
+
+  const updated1xxArr = [];
+
+  initial1xxRecords.forEach(recI => {
+    const updatedRec = updated1xxRecords.filter(recU => recU.id === recI.id);
```

Comment:
If you use `.filter` then you expect a new array with filtered records, I guess in this case the constant should be named as `updatedRecords`. If you expect 1 record then use `.find` instead.